src/GuiHandler.java

```diff
@@ -0,0 +1,373 @@
+        txtTryAWord.setLayoutY(220);
+        txtTryAWord.setLayoutX(80);
+
+        btnTryAWord = new Button();
+        btnTryAWord.setText("LET'S TRY THIS WORD");
+        btnTryAWord.setPrefWidth(150);
+        btnTryAWord.setLayoutY(250);
+        btnTryAWord.setLayoutX(80);
+        btnTryAWord.setOnAction(event -> checkWord());
+        
+        lblLettersCorrect = new Label();
+        lblLettersCorrect.setText("Letters correct: ");
+        lblLettersCorrect.setLayoutY(350);
+        lblLettersCorrect.setLayoutX(10);
+
+        lblLettersWrong = new Label();
+        lblLettersWrong.setText("Letters wrong: ");
+        lblLettersWrong.setLayoutY(380);
+        lblLettersWrong.setLayoutX(10);
+
+        imgHangman = new ImageView();
+        imgHangman.setImage( new Image( getClass().getResource("img/hangman_0.jpg").toString() ) );
+        imgHangman.setFitWidth(110);
+        imgHangman.setPreserveRatio(true);
+        imgHangman.setLayoutY(150);
+        imgHangman.setLayoutX(350);
+        
+        gameEndText = new Label("");
+        gameEndText.setLayoutY(285);
+        gameEndText.setLayoutX(325);
+        gameEndText.setVisible(false);
+        
+        guesWordLenght = new Label("");
+        guesWordLenght.setLayoutY(285);
+        guesWordLenght.setLayoutX(325);
+        guesWordLenght.setVisible(false);
+        
+        reset = new Button("reset");
+        reset.setLayoutY(325);
+        reset.setLayoutX(325);
+        reset.setVisible(false);
+        reset.setOnAction(event -> reset());
+
+        correctLetters = "";
+        
+        //Voeg de controls toe aan de pane
+        pane.getChildren().addAll(lblSetAnswer, txtSetAnswer, btnSetAnswer, shapeLine);
+        pane.getChildren().addAll(lblRules, lblTryALetter, txtTryALetter, btnTryALetter, lblTryAWord, txtTryAWord, btnTryAWord, lblLettersCorrect, lblLettersWrong, imgHangman, gameEndText, reset, guesWordLenght);
+    }
+
+    public Pane getPane()
+    {
+        return pane;
+    }
+
+    /**
+     * Alles voor de logica (Controller)
+     */
+    public void setAnswer()
+    {
+    	if(txtSetAnswer.getText().matches(".*\\d.*")) {errorPopupHandler(0, "", "Het woord mag alleen letters bevatten"); return;}
+    	
+    	int answerLenght = txtSetAnswer.getText().length();
+    	if(answerLenght < 3) 
+    	{
+    		errorPopupHandler(0, "", "Het woord moet minimaal 3 letters hebben");
+    		return;
+    	}
+    	else if(answerLenght > 8)
+    	{
+    		errorPopupHandler(8, "Het woord mag maximaal 8 letters lang zijn", "");
+    		return;
+    	}
+    	
+    	gameRunningStatus = true;
+    	
+        answer = txtSetAnswer.getText().toLowerCase();
+        guesWordLenght.setText("De lengte van het woord is: " + answerLenght);
+        guesWordLenght.setVisible(true);
+        
+        setCorrectLettersLength();
+        
+        lblSetAnswer.setVisible(false);
+        txtSetAnswer.setVisible(false);
+        btnSetAnswer.setVisible(false);
```

Comment:
Kunnen we hier geen functie van maken? 

```suggestion
        DisableAnswerElements(){
             lblSetAnswer.setVisible(false);
             txtSetAnswer.setVisible(false);
             btnSetAnswer.setVisible(false);
        }
```